lua/telescope/_extensions/neorg/find_norg_files.lua

```diff
@@ -18,6 +18,9 @@ local function get_norg_files()
     local current_workspace = dirman.get_current_workspace()
 
     local norg_files = dirman.get_norg_files(current_workspace[1])
+    for i, file in pairs(norg_files) do
+      norg_files[i] = current_workspace[2] .. "/" .. file
```

Comment:
I think it should be use `neorg.configuration.pathsep` for the path separator, instead of `/`. So it adds the specific path for Windows vs other systems.